### ISSUE TITLE
WooCommerce State Missing from Custom Fields

### ISSUE THREAD

**Issue by chrisfox75** [MissVeronica/um-woo-predefined-fields#1]
Installed your plugin when going into my Ultimate Member form editor and i can pick all the Woo Commerce fields except for the State the person lives in, i see billing and shipping city and zipcode along with address etc, but not the State field

Can you help with this?

Thank you

**Comment by MissVeronica** [comment]
Yes you are right. The country codes are incompatible between UM and WooCommerce why I have excluded Country and State.
I will look at this issue if it's possible to get a good solution.

**Comment by chrisfox75** [comment]
I did find out that if i use billing_state in Ultimate Member it does import into the billing field from Woo and if i use the shipping_state it will use the shipping state field, maybe just add the state field to your plugin?

Im not concerned with the country because of it being a selection box on the WP Admin User page

Hope this helps

**Comment by MissVeronica** [comment]
Yes, total integration will include both countries and states.
I have this country/state version working now with Registration and Profile forms but not with the User Account page.
Next I will make country selection optional and only states in a predefined country will be used.

**Comment by MissVeronica** [comment]
Version 2.0.0 ready for download and testing now.

**Comment by MissVeronica** [comment]
Update to Version 2.1.0 for WP User and UM Info of submitted data..

**Comment by MissVeronica** [comment]
Update to Version 2.2.0 for some code improvements

**Comment by MissVeronica** [comment]
Update version 2.3.0
Single country will not require a select field for the Country at Registration/Profile pages and State can be edited in the Account page

**Comment by chrisfox75** [comment]
Thank you for the update can i install this over the old version already installed the same way as the initial install?

**Comment by MissVeronica** [comment]
Yes, WP will remove the older plugin then install the new code
